### ISSUE TITLE
Bug: Unlocking the vault fails with "invalid master password"

### ISSUE THREAD

**Issue by zap0xfce2** [ajrosen/Bitwarden-Accelerator#19]
There is a bug in unlock.sh on line 13:

You have to replace this:

```bash
curl -s -d "password=${p}" "${API}"/unlock | jq -r '.message // .data.title'
```

with this:

```bash
curl -X POST -H "Content-Type: application/json" -d "{\"password\": \"${p}\"}" "${API}"/unlock | jq -r '.message // .data.title'
```

to get unlocking your vault working. If you don't do the change it says invalid master password every time...

**Comment by ajrosen** [comment]
Hmm, it works for me using both payload formats.  The [Vault Management API](https://bitwarden.com/help/vault-management-api/) specifies JSON, so I'll make the change.

I'm curious what version of the Bitwarden CLI you're using?

**Comment by zap0xfce2** [comment]
Using bitwarden-cli 2024.6.0 here. Updated to 2024.8.1 but the bug still persists.

**Comment by ajrosen** [comment]
Fixed in [5.1.1](https://github.com/ajrosen/Bitwarden-Accelerator/releases/tag/bwa-5.1.1)
